src/keri/app/notifying.py

```diff
@@ -353,7 +367,49 @@ def getNotes(self, start=0, end=25):
                 break
 
         return notes
+    
+    def getFltNotes(self, start=0, end=25, rvsd=False, read=None, route=None):
```

Comment:
I don't think this is an appropriate approach for filtering messages in LMDB.  It requires deserializing the data and analyzing it and it hard codes the fields you can search for.  

If you look at the Seeker class in KERIA you'll see an approach to creating LMDB indexes that are more flexible and more performant than this approach.